.github/workflows/dependabotClosePRIssue.yml

```diff
@@ -0,0 +1,26 @@
+name: Dependabot Close Issue
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  issue:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    # Github action permissions: https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v2
+        if: github.event.pull_request.user.login  == 'dependabot[bot]'
+      - name: Open issue if Dependabot PR
```

Comment:
If I'm understanding correctly, I think the name of this job should be "Close issue after Dependabot PR merged" or similar.

Reply:
gah! You caught me. I'll push up a fix. Thanks, Emma!

Reply:
@emmastephenson I'm going to name it "Close issue if Dependabot PR is closed" This job will close the ticket on PR close; that could be a close with a merge or a close without.